src/buildHTML.js

```diff
@@ -716,6 +728,9 @@ groupTypes.op = function(group, options, prev) {
     var supGroup;
     var subGroup;
     var hasLimits = false;
+
+    // TODO(cbreeden) investigate the possibility of putting the supsub
+    //   tages in the atoms themselves.  Allow for easier recursion?
```

Comment:
That was just an idea I was toying with.  I believe the TeX places the scripts inside their respective nodes -- something like if you could imagine each ParseNode had a `superscripts:`, and `subscripts:` field.  The engine will do preliminary processing on the nodes (like changing atom types, etc.), before finally looking for a `scripts` field.  I guess it kind of makes some properties (such as atomtype) more transparent while processing (ie, you don't need to recursively look into the base to determine the atom type).  But I abandoned the idea.  It didn't seem worth it.  Something like this would probably only be useful if other parsenode were refactored to be inline: ie stylechanges.  Where for instance, instead of having a `StyleChange` node wrap the contents it imposed on, you have a single node `StyleChange` which just changes the state of the HTMLRendered.  Anyway, it didn't seem necessary to go down this path, as these recursive definitions seemed quite necessary/natural give the HTML/CSS target.